merlin/schema/io/tensorflow_metadata.py

```diff
@@ -245,17 +253,37 @@ def merlin_properties(feature):
         properties["value_count"] = value_count
         properties["is_list"] = True
         properties["is_ragged"] = value_count.get("min") != value_count.get("max")
+    return {k: v for k, v in properties.items() if not k == "dtype_item_size"}
 
-    return properties
+
+int_dtypes_map = {
+    8: numpy.int8,
+    16: numpy.int16,
+    32: numpy.int32,
+    64: numpy.int64,
+}
+
+
+float_dtypes_map = {
+    16: numpy.float16,
+    32: numpy.float32,
+    64: numpy.float64,
+}
 
 
 def merlin_dtype(feature):
     dtype = None
+    item_size = int(extract_merlin_properties(feature).get("dtype_item_size", 0)) or None
```

Comment:
Rather than extract twice - can we pass the properties extracted by merlin_properties to merlin_dtype? 

Also can we pop the dtype_item_size value from the properties (like we do from is_ragged/is_list) when converting back a merlin schema object?